src/DynamicRupture/Parameters.h

```diff
@@ -74,25 +74,33 @@ inline std::unique_ptr<DRParameters> readParametersFromYaml(std::shared_ptr<YAML
     drParameters->backgroundType = getWithDefault(yamlDrParams, "backgroundtype", 0);
     drParameters->isThermalPressureOn = getWithDefault(yamlDrParams, "thermalpress", false);
     drParameters->t0 = getWithDefault(yamlDrParams, "t_0", 0.0);
-    drParameters->rsF0 = getWithDefault(yamlDrParams, "rs_f0", 0.0);
-    drParameters->rsA = getWithDefault(yamlDrParams, "rs_a", 0.0);
```

Comment:
Is `rs_a` still needed? It doesn't appear in the new parameters

Reply:
I removed this line as rsA is defined as a spatially dependent parameter (yaml file).
(I now also remove the other line mentioning this parameter).